libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.kt

```diff
@@ -49,7 +49,6 @@ import com.salesforce.androidsdk.push.PushMessaging.getRegistrationId
 import com.salesforce.androidsdk.push.PushMessaging.setRegistrationId
 import com.salesforce.androidsdk.push.PushMessaging.setRegistrationInfo
 import com.salesforce.androidsdk.push.PushNotificationsRegistrationChangeWorker.PushNotificationsRegistrationAction
-import com.salesforce.androidsdk.push.PushNotificationsRegistrationChangeWorker.PushNotificationsRegistrationAction.Deregister
```

Comment:
I confirmed this _was_ an unused import.  It wasn't used in the logic on `dev` in this source.  It is used elsewhere, though.